introduction.md

```diff
@@ -0,0 +1 @@
+My name is Tyson. I am a student working towards obtaining a full stack developer certification.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.